### PR TITLE
BFD-826: Fix Patient Search by Part D Contract ID

### PR DESCRIPTION
**Pull request by karlmdavis**
### Change Details

WIP

### External References

- [BFD-xxx](https://jira.cms.gov/browse/BFD-xxx)

### Security Implications

- [ ] new software dependencies

- [ ] altered security controls

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Review by cbrunefearless — DISMISSED** [review]
lgtm

**Review by mackec1 — APPROVED** [review]
WOW! Always good to get better functionality with less code! PatientResourceProvider much better!

File: apps/bfd-server/bfd-server-war/src/main/config/server-start.sh [hunk 1]
```diff
@@ -167,6 +167,7 @@ BFD_PORT="${serverPortHttps}" \
 	BFD_WAR="${warArtifact}" \
 	"${javaHome}/bin/java" \
 	"${maxHeapArg}" \
+	-Dcapsule.jvm.args="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8083" \
```
Comment (jzulim) [inline apps/bfd-server/bfd-server-war/src/main/config/server-start.sh:170]: What's this about?

**Review by sharshar-sb — APPROVED** [review]
Looks good to me.

File: apps/bfd-server/bfd-server-war/src/main/config/server-start.sh (hunk 1 above)
Comment (sb-wnyffenegger) [reply apps/bfd-server/bfd-server-war/src/main/config/server-start.sh:170]: Yeah, is this script used in production? If so it is going to open up a debug port. Not sure that is the desired behavior here.

Comment (karlmdavis) [reply apps/bfd-server/bfd-server-war/src/main/config/server-start.sh:170]: Sorry, this is the "I'm close to done" part. Still working on cleaning that up: want to make sure we keep it around, but disabled, as I've now had to go look up how to do it a few times, and it's getting old. And no, this script is not used in production.

I will be pushing a commit once I get a passing local build.

File: apps/bfd-pipeline/bfd-pipeline-app/src/main/resources/logback.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
 	</appender>
 
 	<!-- At 'debug', Hibernate will log SQL statements. -->
-	<logger name="org.hibernate.SQL" level="info" />
+	<logger name="org.hibernate.SQL" level="debug" />
```
[the remark this answers is not in the document]
Comment (karlmdavis) [reply apps/bfd-pipeline/bfd-pipeline-app/src/main/resources/logback.xml]: No, good catch: thought I'd cleaned all of those up, but missed one, clearly.

[the remark this answers is not in the document]
Comment (karlmdavis) [reply apps/bfd-pipeline/bfd-pipeline-app/src/main/resources/logback.xml]: Oh, actually: I think you were looking at an outdated copy when you reviewed it; I **had** cleaned that up. Can you please refresh the page and verify?

File: apps/bfd-server/bfd-server-war/src/main/config/server-start.sh (hunk 1 above)
Comment (karlmdavis) [reply apps/bfd-server/bfd-server-war/src/main/config/server-start.sh]: Done. Was trying for something fancier, but Bash gets upset if I add empty variables to the server start command for some reason that I'm too lazy to look up.

**Review by sharshar-sb — APPROVED** [review]

**Review by mackec1 — APPROVED** [review]

**Pull request merged**